yarn-completion.bash

```diff
@@ -40,8 +41,12 @@ __yarn_get_package_fields() {
 		cwd="${cwd%/*}"
 	done
 
-	while getopts ":gt:" opt; do
+	while getopts ":gp:t:" opt; do
 		case $opt in
+			p)
+		        cwd=$OPTARG
```

Comment:
No need to set `cwd` here. Just throw `OPTARG` in the `package_dot_json` assignment.